shadows/framework/src/main/java/org/robolectric/shadows/ShadowApplicationPackageManager.java

```diff
@@ -451,11 +451,15 @@ protected PackageInfo getPackageInfo(String packageName, int flags) throws NameN
 
   /**
    * Starting in Android S, this method was moved from {@link android.content.pm.PackageManager} to
-   * {@link ApplicationPackageManager}.
+   * {@link ApplicationPackageManager}. However, it was moved back to {@link
+   * android.content.pm.PackageManager} in T.
    */
   @Override
+  @Implementation(minSdk = S)
   protected PackageInfo getPackageArchiveInfo(String archiveFilePath, int flags) {
-    if (RuntimeEnvironment.getApiLevel() >= S) {
+    int apiLevel = RuntimeEnvironment.getApiLevel();
+    if (apiLevel == S || apiLevel == 32) {
```

Comment:
Don't it work on SDK 33 or AOSP/internal master branch?

Reply:
This is copybara-service not updating the diff properly; this part is supposed to go with #7118

Reply:
I will comment at related PR, Thanks.

Reply:
I found the reason why changing this checking statement at PR 7118. Thanks for pointing out that.